src/Library/demos/Separator/main.blp

```diff
@@ -0,0 +1,90 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Separator";
+  description: _("Separates one widget from another");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Box {
+      orientation: horizontal;
+      halign: center;
+      spacing: 18;
+      margin-bottom: 18;
+      homogeneous: false;
+
+      Box {
+        orientation: vertical;
+        halign: center;
+        spacing: 12;
+
+        Picture picture_one {
+          width-request: 180;
+          height-request: 180;
+        }
+      }
+
+      Separator {
+        margin-start: 6;
+        margin-end: 6;
+      }
+
+      Box {
+        orientation: vertical;
+        halign: center;
+        spacing: 12;
+
+        Picture picture_two {
+          width-request: 180;
+          height-request: 180;
+        }
+      }
+    }
+
+    Separator {
+      margin-bottom: 24;
```

Comment:
```suggestion
```

Same with the these `margin-bottom` properties; much easier to use and adjust the `spacing` property.

This is a good habit to get into, because later in a project it makes it easier to find out where/what properties are responsible for widget layout.

Reply:
yes, I've updated the alignments with spacing property